### PR TITLE
6.get/api/articles/article id/comments

### PR DESCRIPTION
**Pull request by Rawan-Hamza**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -152,3 +152,58 @@ describe("* GET/api/articles/:article_id", () => {
...
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+
+  test("returns comments in date order newist first", () => {
```
Comment (robert112233) [inline __tests__/app.test.js:187]: Great separate test to test sort order 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -152,3 +152,58 @@ describe("* GET/api/articles/:article_id", () => {
...
+  test("returns status code 400 when given invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/zzNdf/comments")
+      .expect(400)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe("bad request");
+      });
+  });
+});
```
Comment (robert112233) [inline __tests__/app.test.js:215]: You are missing a 200 test for a valid article that has no comments, it should return an empty array

File: __tests__/app.test.js [hunk 3]
```diff
@@ -152,3 +152,58 @@ describe("* GET/api/articles/:article_id", () => {
...
+
+  test("returns comments in date order newist first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
```
Comment (robert112233) [inline __tests__/app.test.js:195]: Great sad path tests/error handling 💯

**Review by robert112233 — CHANGES_REQUESTED** [review]
Good work, see feedback and submit another PR for review 👍

**Review by popatre — APPROVED** [review]
Looks good to me
